src/java_bytecode/jar_file.h

```diff
@@ -15,27 +15,27 @@ Author: Daniel Kroening, kroening@kroening.com
 #include <string>
 #include <vector>
 #include <map>
+#include <regex>
+#include <util/message.h>
 
-class jar_filet
+class jar_filet:public messaget
```

Comment:
the JSON parser needs a valid `message_handler` and I also had wanted to give a `status()` about hidden / loaded files (why I will add)